### DISCUSSION TITLE
Child Object Modification Impact on Parent Object's Updated_Timestamp Field

### DISCUSSION THREAD

**Discussion by cilerler** [etsy/open-api#981] [Q&A] [answered]
When a child object undergoes modification, does it affect the parent object's updated_timestamp field?

For example, if a Transaction is inserted, updated, or deleted, will the ShopReceipt object's updated_timestamp field be impacted? Likewise, when a new Image is added to the Images associated with a Listing, does it alter the Listing object's updated_timestamp field?

I aim to establish whether I should query updated_timestamp from specific tables (such as Transactions) or concentrate on main tables (like ShopReceipt).

**Comment by Gareth064** [comment]
Regarding Receipts, the answer is yes and no.
Yes, because once you retrieve the Receipt, its modified timestamp will show that its changed,
No, because when you use the filters for modified timestamp's then it doesn't honor the new timestamps, only the originals.
Have a read through my discussion here #701  (including the ones I linked to) and you will get a feeling of how it works,

As for Listings, I am not sure to be honest. Would be easy to test.

**Comment by cilerler** [comment]
Thank you. I will keep this open for an official response from Etsy Developers, and I hope they will provide an answer.

**Comment by cilerler** [comment, marked as answer]
I've discovered that updating a video path, or presumably other child objects associated with a Listing, doesn't change the updated_timestamp field of the Listing. This might affect how we use this field for data querying.
